website/pages/api-docs/features/filtering.mdx

```diff
@@ -108,12 +73,52 @@ ServiceMeta["foo"]
 
 ### Values
 
-Values are used by matching operators to create an expression. Values can be any valid selector, a number, or a quoted string. For numbers any
-base 10 integers and floating point numbers are possible. For quoted strings,
+Values are used by matching operators to create an expression. Values can be any valid selector, a number, or a string. It is best practice to quote values. 
+
+Numbers can be any
+base 10 integers and floating point numbers are possible. 
+
+When quoting strings,
 they may either be enclosed in double quotes or backticks. When enclosed in
 backticks they are treated as raw strings and escape sequences such as `\n`
 will not be expanded.
 
+## Connect Expressions
```

Comment:
```suggestion
## Connecting Expressions
```
Should this be "connecting" so that its similar to the other headings?

Changing it might also remove any chance that someone misinterprets expressions as only being usable with Consul Connect.

Reply:
Agreed